src/docs/merge-patch.md

```diff
@@ -22,42 +22,42 @@ More information can be found on the [relevant Chromium page](https://chromium.g
 
 ## The merge process
 
-The merge process in the Chromium and V8 tracker is driven by labels in the form of:
+The merge process in the V8 tracker is driven by Attributes in the form of:
 
 ```
-Merge-[Status]-[Branch]
+Merge-Request: 123
+```
+once reviewed, this will be adjusted to
+```
+Merge: Approved-123
+or
+Merge: Rejected-123
 ```
 
-The currently important labels for V8 are:
-
-1. `Merge-Request-{Branch}` initiates the process, and means that this fix should be merged into `{Branch}`. `{Branch}` is the name/number of the V8 branch e.g. `7.2` for M72.
-1. `Merge-Review-{Branch}` means the merge is not approved yet for `{Branch}` e.g. because Canary coverage is missing.
-1. `Merge-Approved-{Branch}` means that the Chrome TPMs have signed off on the merge.
-1. When the merge is done, the `Merge-Approved-{Branch}` label is replaced with `Merge-Merged-{Branch}`.
+After the CL landed
+```
+Merge: Merged-123
+```
 
 ## How to check if a commit was already merged/reverted/has Canary coverage
 
-Use `mergeinfo.py` to get all the commits which are connected to the `$COMMIT_HASH` according to Git.
+Use [chromiumdash](https://chromiumdash.appspot.com/commit/) to verify if the relevant CL has Canary coverage.
 
-```bash
-tools/release/mergeinfo.py $COMMIT_HASH
-```
 
-If it tells you `Is on Canary: No Canary coverage` you should not merge yet because the fix was not yet deployed on a Canary build. A good rule of the thumb is to wait at least 3 days after the fix has landed until the merge is conducted.
+On top the **Releases** section should show a Canary.
 
 ## How to create the merge CL
 
-### Option 1: Using [gerrit](https://chromium-review.googlesource.com/)
+### Option 1: Using [gerrit](https://chromium-review.googlesource.com/) - Recommended
```

Comment:
I actually scripted this process as https://source.chromium.org/chromium/chromium/src/+/main:v8/tools/release/merge_to_branch_gerrit.py, maybe we want to recommend that instead?

Reply:
SG, does it make sense to replace Option 2 below with the new script?

Reply:
sg